src/components/task/TaskWrapper.js

```diff
@@ -0,0 +1,24 @@
+import {CodeEditor} from 'e-stella-code-editor'
+import {tasksApi} from "../../utils/apis/TasksApi";
+import {useParams} from "react-router-dom";
+
+export const TaskWrapper = ({ id: propId, toSolveTask = true, submitLeftOffset = 0 }) => {
+
+    let { id } = useParams()
+
+    if(!id) id = propId
+
+    const fetchTasks = toSolveTask ? () => tasksApi.getTask(id || NaN) : () => new Promise(_ => {})
```

Comment:
Js ma aż taka składnię jsowa?

Reply:
w sensie? Ja się na przykład ostatnio dowiedziałem, że można robić safe checka na wywyołaniu funkcji, więc jak nie wiesz, czy to jest undefined, czy nie, to możesz zrobić `function?.()` XDD

Reply:
To wiedziałem ale ta końcówka wygląda jakbyś obsługiwał że jeśli to null to coś XDD

Reply:
a to tak. Jeśli nie ma taska, to żeby się nie wyjebało, rzucamy czysty kompnenet, bez żadnych testów, descriptiona ani nic